apps/tutorial_servo_control_can/tutorial_servo_control_can/tutorial_servo_control_can.cpp

```diff
@@ -0,0 +1,2 @@
+#include "mbed.h"

```

Comment:
was this added by mistake? if it was, delete it

Reply:
none of the files should ever need to include mbed cuz we force include that header in every file thru cmake

Reply:
okay thank you. I did not know that